compiler/ast/parser.nim

```diff
@@ -996,6 +993,10 @@ proc parseIdentColonEquals(p: var Parser, flags: DeclaredIdentFlags): ParsedNode
       if withPragma in flags: a = p.identWithPragma(allowDot=withDot in flags)
       else: a = parseSymbol(p)
       if a.kind == pnkEmpty: return
+    of tkKeyWords:
```

Comment:
Given that they're doing the same thing, I think it makes sense to merge this with the other branch.

Reply:
Yeah, those were different initially. Only reason I kept them apart was because it highlighted one is "the odd one out" even at this point.